content/03.figures.md

```diff
@@ -5,4 +5,4 @@ The figures can be referenced in the text by using `@fig:label`.
 
 Figure @fig:googletrends shows the interest for "Sci-Hub" and "LibGen" over time.
 
-![Google Trends Search interest for Sci-Hub and LibGen.](https://cdn.rawgit.com/greenelab/scihub/7891082161dbcfcd5eeb1d7b76ee99ab44b95064/explore/trends/google-trends.svg){#fig:googletrends}
+![Google Trends Search interest for Sci-Hub and LibGen.](images/google-trends.svg){#fig:googletrends}
```

Comment:
@dhimmel Yes it can work with url's just need to modify the code to regex check if url.